experimental/LieAlgebras/src/LieAlgebraModule.jl

```diff
@@ -488,7 +534,7 @@ end
 
 Check whether `V` is a direct sum of modules.
```

Comment:
Perhaps mention that this means whether `V` has been constructed as a direct sum, not whether `V` can be decomposed into a direct sum of submodules (same for `is_tensor_product`, `is_exterior_power`, ...)?

Reply:
Changed in 1e40cb835cd2452e348c553d951d4ca2abd8e7e8